test/fixtures/mockUpdate.js

```diff
@@ -0,0 +1,21 @@
+const update0 = require('./g-train-0');
+const update1 = require('./g-train-1');
+const stopInterval = 10
+
+module.exports = function mockUpdate(callCount = 0) {
+  return (req, res) => {
+    const fixture = [update0, update1][callCount % 2];
+    callCount++;
+    const currentTime = Math.round(Date.now() / 1000);
+    const response = fixture.map((trip) => {
+      let firstStopTime = currentTime - stopInterval / 2;
```

Comment:
I have similar thoughts on stop interval.  It can definitely default to 10 but it would be good to give requestors more control and expose it via parameter (since you're worried about it running _too_ quickly)